BoundingBox2D[] bbox_list